### DISCUSSION TITLE
AMS Load Testing

### DISCUSSION THREAD

**Discussion by ghost** [ant-media/Ant-Media-Server#5481] [Q&A]
Hi Team, I want to perform load testing on the ant media server but I am not able to find the testing image locally or on AMS Marketplace.
Can you please provide me with the image?

**Comment by lastpeony** [comment]
You can find the webrtc test tool through this link:
https://gitlab.com/Ant-Media/webrtc-test
Also check this documentation page:
https://antmedia.io/docs/guides/configuration-and-testing/Load-testing/?utm_source=github&utm_medium=wiki#ant-media-webrtc-test-tool
Install AMS as usual and then you can run the test tool and load test your server.
Here is a youtube tutorial showing that:
https://youtu.be/RfQp0A3QIoU?t=1625

If you are asking for actual enterprise version, let us know.
